docs/components/accordion/index.html

```diff
@@ -5,6 +5,7 @@
   components/accordion/test.html: Testing - Accordion
   elements/hx-accordion: <hx-accordion>
   elements/hx-accordion-panel: <hx-accordion-panel>
+  components/stepper/index.html: Stepper
```

Comment:
Our current pattern is to reference components before testing and element documentation.

Please move this above `components/accordion/test.html: Testing - Accordion`.

Reply:
Changes have been done, please review.